build-system/app-index/amphtml-helpers.js

```diff
@@ -87,7 +107,7 @@ const addRequiredExtensionsToHead = (docStr, extensionConf = {
 
   for (let i = 0; i < formTypes.length; i++) {
     const tagName = formTypes[i];
-    if (docStr.search(new RegExp(`\<${tagName}(\s|\>)`))) {
+    if (docStr.search(new RegExp(`\\<${tagName}(\\s|\\>)`)) > -1) {
```

Comment:
This regex was incorrect and so was the usage of `String.search`. Thanks, tests!